src/erlcloud_ddb1.erl

```diff
@@ -141,7 +148,7 @@ create_table(Table, KeySchema, ReadUnits, WriteUnits, Config) ->
             key_schema_json(KeySchema),
             {<<"ProvisionedThroughput">>, [{<<"ReadCapacityUnits">>, ReadUnits},
                                            {<<"WriteCapacityUnits">>, WriteUnits}]}],
-    erlcloud_ddb_impl:request(Config, "DynamoDB_20111205.CreateTable", Json).
+    erlcloud_ddb_impl:request(Config, "DynamoDB_20111205.CreateTable", Json, [{idempotent, false}]).
```

Comment:
isn't this idempotent?

Reply:
Well, not really; if for some reason a previous attempt got to the other end but the response failed to arrive / timed out, should you retry? 
It's not like there's any harm in retrying within a short time window, sure (the table will still be in the state of being created, or even already created) - but getting an "immediate" response along the lines of "Table being created" or "Table already created" sounds, to me, more confusing than getting a network error, retrying, and getting that same error - one can more easily conclude from the latter the reason for the failure is that the request succeeded, only you didn't get the response telling you so on the first attempt.
So, my intention here is mostly about simplifying the interpretation of the response. But you're right in the sense that there's no harm in retrying - for plausible scenarios, anyway.

Reply:
Indeed, there are 2 forms of idempotency here.

CreateTable is idempotent in the sense that you can call it as many times as you like, it will only create it once. In the response sense, it is very much not idempotent—the response when the table exists is an exception which the caller happens.

On the other hand, I very much think the library should let this happen, because you will have to address that error one way or the other, and TBH we have a pretty 'sophisticated' library internally that uses create_table as-is and does exactly this. I think it's robust enough to deal with your library change, but I'm not sure that it's that good in this case.

One of the terrible aspects of this API, for stuff like CreateTable (and even more so, the global table functions), is that the default "first request" timeout is 1 second, unless you explicitly set the timeout in the configuration. @rmpalomino is dealing with that with the November 2019 global tables API updates right now.

Reply:
> One of the terrible aspects of this API, for stuff like CreateTable (and even more so, the global table functions), is that the default "first request" timeout is 1 second [...]

Wow, I didn't know that! Yes, it looks way too low.